.github/workflows/validate.yml

```diff
@@ -52,6 +52,14 @@ jobs:
           path: ${{ steps.set-puppeteer-cache.outputs.PUPPETEER_CACHE_DIR }}
           key: puppeteer-${{ runner.os }}-${{ hashFiles('./pnpm-lock.yaml') }}
 
+      - name: Cache babel-loader
+        id: babel-loader-cache
+        uses: actions/cache@v3
```

Comment:
Worth bumping to v4? Or maybe doing that across the board separately?

Reply:
Was going to do it separately 👍